sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -87,6 +87,27 @@ internal static LusidInstrument CreateExampleBond()
                 instrumentType: LusidInstrument.InstrumentTypeEnum.Bond
             );
 
+        internal static LusidInstrument CreateExampleZeroCouponBond()
+            => new Bond(
+                startDate: new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero),
+                maturityDate: new DateTimeOffset(2020, 9, 18, 0, 0, 0, TimeSpan.Zero),
+                domCcy: "USD",
+                principal: 100m,
```

Comment:
what units and cost should transactions/holdings be booked with (its not clear from this example)?

Reply:
The ZCB creation is more-or-less identical to a bond creation example we already had before. The only _difference_ is that there is a single payment and coupon rate is zero. The asked question (if clarification is requited) should not be answered in the ZCB example, as other instruments use the same  **domCcy**, **principal** parameters.